proto/process/agent.proto

```diff
@@ -93,6 +102,8 @@ message CollectorContainer {
 
 	// Post-resolved fields
 	Host host = 8;
+
```

Comment:
Can we mark fields 6 & 7 as reserved if we aren't using them?

Reply:
I'm not sure whether they are used or not, grepping datadog-agent repo doesn't seem like it. I'll work on agent code first and come back to this clean up later.

Reply:
Fields 6 and 7 are still used by older agents and our tag handling code parses them so we'll have to keep them around for now unfortuantely.